src/app/portlets/content-types/edit/content-types-edit.component.ts

```diff
@@ -81,7 +83,10 @@ export class ContentTypesEditComponent implements OnInit {
                 'contenttypes.form.identifier',
                 'contenttypes.dropzone.rows.add'
             ])
-            .subscribe();
+            .subscribe(() => {
```

Comment:
Can you check why do we ask for all this:

```
                'contenttypes.action.create',
                'contenttypes.action.edit',
                'contenttypes.action.form.cancel',
                'contenttypes.action.update',
                'contenttypes.content.content',
                'contenttypes.content.create.contenttype',
                'contenttypes.content.edit.contenttype',
                'contenttypes.content.fileasset',
                'contenttypes.content.form',
                'contenttypes.content.htmlpage',
                'contenttypes.content.key_value',
                'contenttypes.content.persona',
                'contenttypes.content.vanity_url',
                'contenttypes.content.variable',
                'contenttypes.content.widget',
                'contenttypes.form.identifier',
                'contenttypes.dropzone.rows.add'
```

I don't think we're using all that in this component.